packages/block-editor/src/components/media-placeholder/index.js

```diff
@@ -288,7 +288,7 @@ export function MediaPlaceholder( {
 		}
 	}
 
-	async function onHTMLDrop( HTML ) {
+	async function onMediaDrop( HTML ) {
 		const blocks = pasteHandler( { HTML } );
 		return await handleBlocksDrop( blocks );
 	}
```

Comment:
Nit - it might good to make it consistent with the docs for `onMediaDrop` (though my second comment would make this unnecessary)
```suggestion
	async function onMediaDrop( media ) {
		const blocks = pasteHandler( { media } );
		return await handleBlocksDrop( blocks );
	}
```